packages/suite/src/components/suite/notifications/NotificationRenderer/NotificationRenderer.tsx

```diff
@@ -185,6 +185,8 @@ export const NotificationRenderer = ({
             );
         case 'coin-scheme-protocol':
             return <CoinProtocolRenderer render={render} notification={notification} />;
+        case 'local-first-sync-keys-error':
```

Comment:
```suggestion
        case 'suite-sync-keys-error':
```

Lets use  new correct naming for new code.